packages/manager/modules/bm-server-components/src/ipmi/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,71 @@
+{
+  "server_configuration_impi_loading_error": "Une erreur est survenue lors de la récupération des informations",
+  "server_configuration_impi_reset_interfaces": "Redémarrer IPMI",
+  "server_configuration_impi_reset_sessions": "Supprimer les sessions",
+  "server_configuration_impi_off": "Le module IPMI n'est pas disponible pour ce serveur.",
+  "server_configuration_impi_off_kvm": "Toutefois, si vous souhaitez bénéficier de cette fonctionnalité, vous pouvez utiliser un KVM.",
+  "server_configuration_impi_order_kvm": "Commander un KVM",
+  "server_configuration_impi_disabled": "Le statut du serveur est <b>{{t0}}</b>. Le module IPMI est donc désactivé.",
+  "server_configuration_impi_test": "Tester IPMI",
+  "server_configuration_impi_http": "Test d'accès au module IPMI (http)",
```

Comment:
```suggestion
  "server_configuration_impi_http": "Test d'accès au module IPMI (HTTP)",
```

Reply:
Good suggestion. This is legacy code. I have just moved/split it to new file and module. The aim of the refactoring is to do with minimal code changes. This can be taken as a separate task.